protocol/synthetix/storage.dump.sol

```diff
@@ -417,6 +417,8 @@ library AccountRBAC {
     bytes32 internal constant _PERPS_MODIFY_COLLATERAL_PERMISSION = "PERPS_MODIFY_COLLATERAL";
     bytes32 internal constant _PERPS_COMMIT_ASYNC_ORDER_PERMISSION = "PERPS_COMMIT_ASYNC_ORDER";
     bytes32 internal constant _BURN_PERMISSION = "BURN";
+    bytes32 internal constant _PERPS_PAY_DEBT_PERMISSION = "PERPS_PAY_DEBT";
+    bytes32 internal constant _PERPS_SPLIT_ACCOUNT_PERMISSION = "PERPS_SPLIT_ACCOUNT";
```

Comment:
Can we prefix this with `_BFP_xxx` as this is bfp-market specific?

Reply:
happy to do this if you want. Just a question if we want other perps market to potentially reuse them

Reply:
I don't think it would be used in other perp markets, it's very bfp-market specific tbh but I'm ok if you want to keep it named the same.

Reply:
Added BFP